src/main/docker/Dockerfile

```diff
@@ -14,40 +14,52 @@
 # limitations under the License.
 #
 
+# Preliminary image that includes Elasticsearch and related files
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.3.2 AS builder
```

Comment:
I don't like calling this a preliminary builder image when it is just a single `FROM` statement and no building up of anything happens.

Let's change the comment to:
`# Reference the official Elasticsearch 6 image`

And instead of `builder` use `es6-image` or similar.